corehq/apps/userreports/es/adapter.py

```diff
@@ -228,6 +232,6 @@ def build_es_mapping(data_source_config):
         if datatype == 'string':
             properties[indicator['column_id']]['index'] = 'not_analyzed'
     mapping = deepcopy(UCR_INDEX_SETTINGS)
-    mapping.update(data_source_config.get_es_index_settings())
+    mapping['settings'].update(data_source_config.get_es_index_settings()['settings'])
```

Comment:
this was necessary because it'd overwrite settings defined elsewhere? i don't think i wrote this portion